site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.scss

```
@@ -165,6 +166,23 @@
   }
 }
 
+.source-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: translate(25%, -25%);
+  width: 20px;
+  height: 20px;
+  border-radius: 50%;
```

In the case that a similar pill is not the same width or height, this would cause it to be an ellipse rather than a pill. instead, use a fixed amount that is no less than half the height of the element:
```suggestion
  border-radius: 20px;
```